### ISSUE TITLE
How to silence console.errors from the SDK?

### ISSUE THREAD

**Issue by ethanneff** [splitio/javascript-client#335]
**Problem**

`console.error()` interrupts React Native apps with a large overlay. This is problematic when debugging the app in offline mode. 

**Details**

I get the following error from the Split SDK when implementing a factory:

```
console.error: "[ERROR] Split SDK emitted SDK_READY_TIMED_OUT event."
```

```ts
const factory = SplitFactory({
  core: { authorizationKey, key }
});
```

![Screenshot_20191231-200804](https://user-images.githubusercontent.com/2933593/71637554-69d43a00-2c0b-11ea-8861-a9a23cab7d62.png)

**Question**

How do I silence `console.errors` from the Split SDK? Even when listening to the event, the `console.error` still comes through.

```ts
client.on(client.Event.SDK_READY_TIMED_OUT, function () {});
```

**Comment by NicoZelaya** [comment]
Hi @ethanneff! 

I would like to start by offering you my most sincere apologies for dropping the ball on getting back to you, as well as ours for the inconveniences.

It's actually our plan to remove the calls to `console.error` and just stick with `console.log` method. We'll also treat these as just another log level instead of always printing.
While it's not an actual error or exception of any kind and it doesn't affect the execution flow (should only be a red message) we've recently learned that some client side tools might consider these as actual errors.

I'll make sure to speed up resolution for this one so we can test and make a patch release before end of week. 

Kindest regards,
Nicolas Zelaya.

**Comment by ethanneff** [comment]
When was this fixed? Our app is currently using `10.10.0` and we are still getting the errors. #351 did not fix it. 

![Simulator Screen Shot - iPhone 11 - 2020-03-20 at 15 08 41](https://user-images.githubusercontent.com/2933593/77216078-10c2c680-6ad5-11ea-89f1-8b9a5e2fa65c.png)

**Comment by NicoZelaya** [comment]
Hi @ethanneff ! Sorry I didn't want to tag you earlier to not generate a notification because I thought we reached out separately but yes, should be fixed. Can you try version `10.11.1` ? 

I'm curious why you say PR #351 didn't solve it, how are you testing that? Was included on the `10.11.0` release if I'm not mistaken.

In any case, give a try to `10.11.0` or `10.11.1` and you should be able to turn off error logs and besides, we're not using the `console.error` method anymore in order to fix these React Native issue.

I'll reopen until I hear back from you.
Thanks in advance!
Nico.
